### DISCUSSION TITLE
Response Not Ready paramter RD Exponenet.

### DISCUSSION THREAD

**Discussion by RohitPatil555** [DMTF/libspdm#2380] [General]
https://github.com/DMTF/libspdm/blob/6a66f71cef02a0a4d514aeb38420754fcebcb28a/library/spdm_responder_lib/libspdm_rsp_handle_response_state.c#L41C44-L41C44

I was going through code and found this parameter as hard coded in code. That is not seen correct.

I think as like CT exponent in CAPABILITY message, this parameter also need to be configurable from context or #define as configuration parameter.

Please let me know which approach community prefer for this parameter, it will be better if get to know reason behind it.

**Comment by steven-bellock** [comment]
This is captured in #1983. But, in general, for a Responder implementation it's way easier to not support `ResponseNotReady` and just have the response ready. Then the Responder doesn't have to deal with pre-emption and timers and whatnot. That's probably why implementers haven't been pushing for it.
